README.md

```diff
@@ -201,12 +201,14 @@ The [`scripts/presubmit.sh`](scripts/presubmit.sh) script runs various tools
 and tests over the codebase.
 
 ```bash
-# Install various codebase checkers
-go get -u github.com/client9/misspell/cmd/misspell
-go get -u github.com/fzipp/gocyclo
-go get -u github.com/gordonklaus/ineffassign
-go get -u github.com/golang/lint/golint
-# Run checks, tests and code generation
+# Install gometalinter and all linters
+go get -u github.com/alecthomas/gometalinter
+gometalinter --install
+
+# Run linters
+gometalinter --config=gometalinter.json ./...
```

Comment:
If someone follows these instructions as-is, are they going to run the linters twice?

If so, could move this down below the `presubmit` invocation with a comment:
```
# Or just run the linters alone:
gometalinter --config=gometalinter.json ./...
```

Reply:
Done.